notifications/16102019-tutorialOffer.yml

```diff
@@ -0,0 +1,9 @@
+requirements:
+    local_config:
+        - startup/launch-count<5
+icon: "canvas/icons/announcement.png"
+title: "Orange tutorials"
+text: "Do you want to learn how to use Orange? Check our video tutorials."
```

Comment:
Consider changing this to:

`New to Orange? Get started by checking out our video tutorials!`

Reply:
That is better.